source/elements/oneMath/source/appendix/acknowledgment.inc.rst

```diff
@@ -2,10 +2,10 @@
 ..
 .. SPDX-License-Identifier: CC-BY-4.0
 
-.. _onemkl_acknowledgment:
+.. _onemath_acknowledgment:
 
 Acknowledgment
 ---------------
-The oneMKL `Technical Advisory Board <https://github.com/oneapi-src/oneAPI-tab>`__ members
```

Comment:
Since the oneMKL TAB was the precursor to the Math SIG, perhaps this would be better phrased as:

The Math `Special Interest Group <https://github.com/uxlfoundation/foundation/tree/main/math>`__ members ...

Reply:
Thank you for your review @sknepper 
I updated it in https://github.com/uxlfoundation/oneAPI-spec/pull/596/commits/ddfe85b9df2274d8c44191fed4151b50f7d5c538